lib/ansible/modules/windows/win_blockinfile.py

```diff
@@ -0,0 +1,150 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2017, Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: win_blockinfile
+short_description: Ensure a particular line is in a file, or replace an existing line using a back-referenced regular expression
+version_added: '2.8'
+description:
+- This module will insert/update/remove a block of multi-line text surrounded by customizable marker lines.
+options:
+  path:
+    description:
+      - The path of the file to modify.
+      - Note that the Windows path delimiter C(\) must be escaped as C(\\) when the line is double quoted.
```

Comment:
It's not a specific advice for this parameter, or this module.
So I am not a big fan of stating this here. It would mean we have to document it like this for every module.
We have this in the Windows documentation.